package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.2",
   "description": "Apollo Server plugin that adds New Relic Node.js agent instrumentation.",
   "main": "./index.js",
+  "types": "./index.d.ts",
```

Comment:
I think you also have to add this to the files array because when this module is published this file
will be ignored